frontend/root_words_ui/pages/index.js

```diff
@@ -1,56 +1,29 @@
 import Head from "next/Head";
```

Comment:
I'm not sure why this was working before, but the import should be from `next/head` not `next/Head`

Reply:
Edit: I handled this issue in [this commit](https://github.com/Quran-Journey/documents/pull/27/commits/dedec8388f166b5eada0bc8683f62a450af84f73)